OmiseSDK/Helpers/Bundle+OmiseSDK.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+private final class OmiseSDK216d048c5feca16469a7a71566256537 {
```

Comment:
Is it worth adding a comment explaining why the class is named like this?

Reply:
Don't think so, the next line of code use it and for iOS developer its pretty straight forward